platforms/gke/base/core/container_cluster/container_cluster.tf

```diff
@@ -200,33 +171,13 @@ resource "google_container_cluster" "cluster" {
 
   node_pool {
     initial_node_count = 1
-    name               = "system"
-
-    autoscaling {
-      location_policy      = "BALANCED"
-      total_max_node_count = 32
-      total_min_node_count = 1
-    }
-
-    network_config {
-      enable_private_nodes = true
-    }
-
+    name               = "default-pool"
     node_config {
```

Comment:
Is it possible to add variables to specify custom labels and taints for this node pool? This will be useful for the federated learning use case.

Reply:
We can close this one as well because of the default node pool removal.